Scripts/travis.sh

```diff
@@ -108,7 +108,9 @@ elif [ "${TYPE}" == "EXAMPLE_PODS_SWIFT" ]; then
   cd gem/
   bundle install
   rake install
-  cd ..
+  cd ../Demo/EarlGreyExample
+  pod update
```

Comment:
if this is on travis, are you sure Travis won't clean up after it's done?

also you can consider `pod update --project-directory=` after `pod install`